jackal_base/include/jackal_base/sensor_frame_node.h

```diff
@@ -0,0 +1,68 @@
+/*
+ *  Copyright (C) 2021, The University of Texas at Austin
+ *  Robert Blake Anderson <blakeanderson@utexas.edu>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
For legal reasons we cannot accept PRs that contain GPL-licensed code.  You're welcome to maintain your own fork that is dual-licensed, but the GPLv3 is unfortunately not compatible with our software releases.
By default we use the BSD 3-clause license for all open-source Clearpath ROS packages.

Reply:
This message had slipped by me, but I see it now since we're discussing that other issue. After looking into it, I am allowed to re-license this under BSD. Expect a commit shortly.